src/anypytools/tools.py

```diff
@@ -347,6 +347,17 @@ def from_shelve(cls, filename, key='results'):
         out = db[key]
         db.close()
         return out
+      
+    def to_json_list(self, ):
+        listofdicts_output = []
+        for AnyPyProOut in self:
+            trial = {}
+            for key, val in AnyPyProOut.items():
+                if isinstance(val, np.ndarray):
+                    val = val.tolist()
+                trial[key] = val
+            listofdicts_output.append(trial)
+        return listofdicts_output
```

Comment:
Nice work. With respect to our new style guide there are few things here. 

1. Function should have a doc string.
2. No comma after self.
3. Function doesn't return a json object so maybe name it `tolist()` instead. 
4. `AnyPyProOut` is Capitalized Words style. Use lower case name for variables. E.g. `for elem in self`

```` 
def tolist(self):
    """ Return as native python types (list of dicts)"""
        new_list= []
        for elem in self:
            new_elem= {}
            for key, val in elem.items():
                if isinstance(val, np.ndarray):
                    val = val.tolist()
                new_elem[key] = val
            new_list.append(new_elem)
        return new_list

````
But in general, when transforming list or dictonaries it can sometimes be more readable if you write it as a list comprehension. 

```python
def tolist():
    """ Return as native python types (list of dicts)"""
    return [
        { 
            k: v.tolist() if isinstance(v, np.ndarray) else v
            for k, v in elem.items()
        }
        for elem in self
    ]
```